readme.markdown

```diff
@@ -261,6 +261,17 @@ To run tests a single time, simply run the following command. This is used in CI
 npm test
 ```
 
+# Troubleshooting
```

Comment:
I would get rid of this heading, let's change `### CSS Classes` into `## CSS`, and place it right under `drake.destroy()` _(so, below the API section)_